style.css

```diff
@@ -0,0 +1,371 @@
+    font-size: 36px;
+    background-color: #FFFFFF;
+}
+
+.fa-google {
+    background: #dd4b39;
+    color: white;
+}
+
+.fa-linkedin {
+    background: #327BB0;
+    color: white;
+}
+
+img {
+    max-width: 100%;
+    margin: 0 auto;
+    padding: 5px;
+}
+
+.photo {
+  width: 80%;
+  margin: 0 auto;
+  clear: both;
+}
+
+.about {
+    line-height: 2rem;
+    font-family: 'Roboto Condensed', sans-serif;
+    padding: 10px 15px 10px 15px;
+}
+
+.aboutme {
+    padding: 0px 15px 0px 15px;
+    line-height: 2rem;
+    font-family: 'Roboto Condensed', sans-serif;
+}
+
+.project1 {
+    padding: 5px 5px 5px 5px;
+    margin: 0 auto;
+    font-family: 'Roboto Condensed', sans-serif;
+    font-weight: bold;
+    clear: both;
+    border: 1px solid black;
+}
+
+.w1d2-photolink {
+    padding: 15px 0px 0px 0px;
+}
+
+.w1d1-photolink {
+    padding: 15px 0px 0px 0px;
+}
+
+.w1d1-text, .w1d2-text {
+    padding: 0px 15px;
+}
+
+.project2 {
+    font-family: 'Roboto Condensed', sans-serif;
+    font-weight: bold;
+    padding: 5px 5px 5px 5px;
+    border: 1px solid black;
+    margin: 10px 0px;
+
+}
+
+.resume {
+    clear: both;
+
+}
+
+.workexperience {
+    padding: 0px 15px 10px 15px;
+}
+
+.licensure {
+    max-width: 100%;
+}
+
+.licensure-header {
+    display: inline;
+    float: left;
+    max-width: 45%;
+    font-size: 1.5em;
+    -webkit-margin-before: 0.83em;
+    -webkit-margin-after: 0.83em;
+    -webkit-margin-start: 0px;
+    -webkit-margin-end: 0px;
+    font-weight: bold;
+}
+
+.licensescont {
+    float: left;
+    max-width: 68%;
+    padding: 25px 0px 0px 15px;
+}
+
+.workexperience-header {
+    clear: both;
+}
+
+.skillslist {
+    margin: 0 auto;
+    align-content: justify;
+    padding: 5px 25px;
+}
+
+
+.education-degree {
+    font-size: 19px;
+}
+
+.orgs {
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: row;
+    margin: 0 auto;
+    padding: 10px;
+}
+.org_item {
+    list-style-type: none;
+    width: 45%;
+    padding: 5px;
+    
+}
+
+.masters {
+    padding: 10px 15px;
+}
+
+
+.footer {
+    background-color: #69027B;
+    padding: 30px 0px;    
+}
+
+.footer_menu {
+    background-color: #69027B;
+    height: 100px;
+    display: inline;
+}
+
+.footermenu_item {
+    background-color: #69027B;
+    max-width: 15%;
+    padding: 10px 15px 10px 15px;
+    float: left;
+    margin: 5px;
+    color: white;
+    font-family: 'Roboto Condensed', sans-serif;
+    font-size: 18px;
+}
+a.footermenu_item {
+    text-decoration: none;
+}
+
+a.footermenu_item:hover {
+    text-decoration: underline;
+}
+
+.footersocial-menu {
+    max-width: 30%;
+    display: inline;
+    text-align: right;
+    /* padding: 5px; */
+    margin: 0px 10px 0px 20px;
+    font-size: 36px;
+    background-color: #69027B;
+}
+
+.footersocial_item {
+    max-width: 30%;
+    float: right;
+    padding: 1px 1px;;
+    margin: 0px 20px;
+    font-size: 36px;
+    background-color:white;
+}
+
+
+@media screen and (max-width: 798px) {
```

Comment:
Just to note, between this line and the next media query, both sets of CSS will apply if you're at exactly 798px. I notice you're using both max-width and min-width: in general, you will be better off using one or the other.

Reply:
Thank you for the feedback! It is definitely helpful, and I really appreciate you taking the time to so thoroughly review the code and provide the feedback!